nvflare/fuel/f3/sfm/conn_manager.py

```diff
@@ -139,7 +139,7 @@ def stop(self):
                 connector.stopping = True
                 connector.driver.shutdown()
 
-        self.conn_mgr_executor.shutdown(True)
+        self.conn_mgr_executor.shutdown(False)
```

Comment:
I noticed you shutdown the socket now, so maybe this can stay "True"

Reply:
Yeah. Changed to true.